Formula/pacparser.rb

```diff
@@ -0,0 +1,28 @@
+class Pacparser < Formula
+  desc "Library to parse proxy auto-config (PAC) files"
+  homepage "https://github.com/pacparser/pacparser"
+  url "https://github.com/pacparser/pacparser/archive/1.3.6.tar.gz"
+  sha256 "c1f6f2ccd4fec329175f882102e47c57328e8e6e16aa9dbd4f8b859f9a028e83"
+  head "https://github.com/pacparser/pacparser.git"
+
+  depends_on "python" => :optional
+
+  def install
+    # Disable parallel build due to upstream concurrency issue.
+    # https://github.com/pacparser/pacparser/issues/27
+    ENV.deparallelize
+    system "make", "-j1", "-C", "src", "install",
```

Comment:
Can remove the `-j1` now. Additionally, a `Dir.chdir "src"` will save needing to repeat it below. You may also want to use a `ENV["VERSION"] = version` to avoid setting that twice, too.